src/vos/tests/vos_dfs_sample.yaml

```diff
@@ -1,34 +1,111 @@
 ---
 # Sample conflig file DFS files and directories
 num_pools: 1000
-time_key: &time
-  count: 3
-  size: 5
+
+dfs_magic: &magic
+  type: integer
   overhead: meta
   value_type: single_value
-  values: [{"count": 3, "size": 8}]
+  values: [{"count": 1, "size": 8}]
 
-oid_key: &oid
-  size: 3
+dfs_sb_version: &sb_ver
+  type: integer
   overhead: meta
   value_type: single_value
-  values: [{"size": 16}]
+  values: [{"count": 1, "size": 2}]
 
-mode_key: &mode
-  size: 4
+dfs_sb_layout_version: &layout_ver
+  type: integer
+  overhead: meta
+  value_type: single_value
+  values: [{"count": 1, "size": 2}]
+
+dfs_sb_feat_compat: &compat
+  type: integer
+  overhead: meta
+  value_type: single_value
+  values: [{"count": 1, "size": 8}]
+
+dfs_sb_feat_incompat: &incompat
+  type: integer
   overhead: meta
   value_type: single_value
-  values: [{"size": 4}]
+  values: [{"count": 1, "size": 8}]
+
+dfs_sb_mkfs_time: &mkfs
+  type: integer
+  overhead: meta
+  value_type: single_value
+  values: [{"count": 1, "size": 8}]
+
+dfs_sb_state: &state
+  type: integer
+  overhead: meta
+  value_type: single_value
+  values: [{"count": 1, "size": 8}]
+
+dfs_chunk_size: &def_chunk_size
+  type: integer
+  overhead: meta
+  value_type: single_value
+  values: [{"count": 1, "size": 8}]
+
+dfs_obj_class: &obj_class
+  type: integer
+  overhead: meta
+  value_type: single_value
+  values: [{"count": 1, "size": 8}]
+
+dfs_sb_metadata: &sb_metadata
+  count: 1
+  type: integer
+  overhead: meta
+  akeys: [*magic, *sb_ver, *layout_ver, *compat, *incompat, *mkfs, *state, *def_chunk_size, *obj_class]
+
+
+mode_t: &mode
+  count: 1
+  size: 4
+
+obj_id_t: &oid
+  count: 1
+  size: 16
+
+time_t: &time
+  count: 3
+  size: 8
+
+daos_size_t: &chunk_size
+  count: 1
+  size: 8
+
+symlink_value: &symlink
+  count: 1
+  size: 1
+
+# The dfs_entry struct has a size of 64 bytes
+p_t: &padding
+  count: 1
+  size: 11
+
+dfs_inode: &dfs_inode
+  type: integer
+  overhead: meta
+  value_type: array
+  values: [*mode, *oid, *time, *chunk_size, *symlink, *padding]
```

Comment:
We can replace it with:
`values: [{"count": 1, "size": 64}]`
But that will hide the padding and the meaning of the individual values

Reply:
Is this right?   The inode is contiguous so it should really be one extent that encapsulates the size of all of the entries together.   Otherwise, we are counting overheads as if we insert a unique value for each attribute and never aggregate them into a single extent

Reply:
No it isn't. I realized that 
`values: [{"count": 1, "size": 64}]`
is not equal to
`values: [*mode, *oid, *time, *chunk_size, *symlink, *padding]`

Reply:
so it needs to be the former.   Otherwise, we are attributing more metadata to the dfs_inode than would be typical.

Reply:
I am confused. Here are three quick experiments using different values at line 93.

`values: [{"count": 1, "size": 64}]` gives us:
Metadata totals:
        pool                :     273.44 K ( 0.00%)
        container           :     671.88 K ( 0.01%)
        object              :     344.08 M ( 2.93%)
        dkey                :     422.96 M ( 3.60%)
        akey                :     564.58 M ( 4.80%)
        single_value        :     132.56 M ( 1.13%)
        array               :       2.33 G (20.26%)
        total_meta          :       3.76 G (32.72%)
        user_meta           :      15.26 M ( 0.13%)
        user_value          :       7.71 G (67.15%)
        scm_total           :       3.85 G (33.56%)
Total bytes with user data:      11.48 G

`values: [{"count": 1, "size": 4}, {"count": 1, "size": 16}, {"count": 3, "size": 8}, {"count": 1, "size": 8}, {"count": 1, "size": 1}, {"count": 1, "size": 11}]` gives us

Metadata totals:
        pool                :     273.44 K ( 0.00%)
        container           :     671.88 K ( 0.01%)
        object              :     344.08 M ( 2.85%)
        dkey                :     422.96 M ( 3.50%)
        akey                :     564.58 M ( 4.67%)
        single_value        :     132.56 M ( 1.10%)
        array               :       2.64 G (22.37%)
        total_meta          :       4.07 G (34.50%)
        user_meta           :      15.26 M ( 0.13%)
        user_value          :       7.71 G (65.37%)
        scm_total           :       4.17 G (35.32%)
Total bytes with user data:      11.80 G

`values: [*mode, *oid, *time, *chunk_size, *symlink, *padding]` gives us:

Metadata totals:
        pool                :     273.44 K ( 0.00%)
        container           :     671.88 K ( 0.01%)
        object              :     344.08 M ( 2.85%)
        dkey                :     422.96 M ( 3.50%)
        akey                :     564.58 M ( 4.67%)
        single_value        :     132.56 M ( 1.10%)
        array               :       2.64 G (22.37%)
        total_meta          :       4.07 G (34.50%)
        user_meta           :      15.26 M ( 0.13%)
        user_value          :       7.71 G (65.37%)
        scm_total           :       4.17 G (35.32%)
Total bytes with user data:      11.80 G

So, if I understood correctly, the correct implementation is `values: [{"count": 1, "size": 64}]` and we should get rid of lines 64-87. Is it correct?

Reply:
correct